post_installation_configuration/coreos-layering.adoc

```diff
@@ -99,6 +106,7 @@ After you create the machine config, the MCO:
 [IMPORTANT]
 ====
 It is strongly recommended that you test your images outside of your production environment before rolling out to your cluster.
+Additionally, the minimum build environment expected is RHEL 8.7 with podman 4.2.0, however your containers might build in other environments without issue.
```

Comment:
I'd be curious to know a little bit more about this.  What was the problem?  Can we track issues with other build environments somewhere?

Reply:
For some reason on RHEL 8.6 with podman 4.0.2 the build fails to run dracut:
```
Installing 4 packages:
  kernel-4.18.0-372.40.1.el8_6.iavf.bz2149746.bz2152493.x86_64 (@commandline)
  kernel-core-4.18.0-372.40.1.el8_6.iavf.bz2149746.bz2152493.x86_64 (@commandline)
  kernel-modules-4.18.0-372.40.1.el8_6.iavf.bz2149746.bz2152493.x86_64 (@commandline)
  kernel-modules-extra-4.18.0-372.40.1.el8_6.iavf.bz2149746.bz2152493.x86_64 (@commandline)
Updating: kernel-core;4.18.0-372.40.1.el8_6.iavf.bz2149746.bz2152493;x86_64;local
Updating: kernel-modules;4.18.0-372.40.1.el8_6.iavf.bz2149746.bz2152493;x86_64;local
Updating: kernel;4.18.0-372.40.1.el8_6.iavf.bz2149746.bz2152493;x86_64;local
Updating: kernel-modules-extra;4.18.0-372.40.1.el8_6.iavf.bz2149746.bz2152493;x86_64;local
Cleanup: kernel;4.18.0-372.40.1.el8_6;x86_64;installed
Cleanup: kernel-modules-extra;4.18.0-372.40.1.el8_6;x86_64;installed
Cleanup: kernel-modules;4.18.0-372.40.1.el8_6;x86_64;installed
Cleanup: kernel-core;4.18.0-372.40.1.el8_6;x86_64;installed
error: Input/output error (os error 5)
COMMIT side-kernel-4.18.0-372.40.1.el8_6.iavf.bz2149746.bz2152493.x86_64:latest
--> 2e0ad5807ff
Successfully tagged quay.io/midu/side-kernel-4.18.0-372.40.1.el8_6.iavf.bz2149746.bz2152493.x86_64:latest
2e0ad5807ff7896a609479051067eebbca2597106ac5d98d30129d23869db5d7
```

The weird part is:
```
error: Input/output error (os error 5)
```

Reply:
That seems highly likely to be an issue with the storage stack on that particular machine.  Do other container builds fail in the same way for example?

Reply:
Well I just replicated on a VM. Not all fail the same way if I take out the `rpm-ostree cliwrap install-to-root /` it does not happen.

Reply:
Let's file a bug...somewhere?  Dunno, against rpm-ostree to start?

Reply:
I created this to start: https://github.com/coreos/rpm-ostree/issues/4261